src/codeu/chat/client/commandline/Chat.java

```diff
@@ -14,12 +14,10 @@
 
 package codeu.chat.client.commandline;
 
-import codeu.chat.client.core.Context;
-import codeu.chat.client.core.ConversationContext;
-import codeu.chat.client.core.MessageContext;
-import codeu.chat.client.core.UserContext;
+import codeu.chat.client.core.*;
```

Comment:
No wildcard imports. I think this has been resolved in a later pull request though.